go.mod

```diff
@@ -12,20 +12,18 @@ replace (
 
 require (
 	github.com/ajeddeloh/go-json v0.0.0-20170920214419-6a2fe990e083 // indirect
-	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e // indirect
 	github.com/coreos/ignition v0.33.0
 	github.com/go-logr/zapr v0.1.0
-	github.com/gogo/protobuf v1.2.1 // indirect
 	github.com/spf13/cobra v0.0.5
-	github.com/spf13/pflag v1.0.3
-	github.com/stretchr/testify v1.3.0
+	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.4.0
```

Comment:
Why are you bumping these?

Reply:
They are getting updated by `go mod tidy`.
I verified by checking out upstream master go mod and go sum, then running `go get sigs.k8s.io/controller-runtime/pkg/log/zap` and `go mod tidy`